packages/material/src/complex/MaterialTableControl.tsx

```diff
@@ -33,7 +33,7 @@ const EmptyTable = () => (
 );
 
 const TableHeaderCell = ({cellProperty}) =>
-  <TableCell >{cellProperty}</TableCell>;
+  <TableCell >{_.capitalize(cellProperty)}</TableCell>;
```

Comment:
Does this fix the Label issue from #894?

Reply:
Yes.